src/opm/input/eclipse/EclipseState/Aquifer/Aquancon.cpp

```diff
@@ -178,16 +179,25 @@ namespace Opm {
                                     add_cell(aquanconKeyword.location(), work, grid, aquiferID, global_index, influx_coeff, influx_mult, faceDir);
                                 }
                             } else {
+                                ++num_connections_to_inactive_cells;
                                 const auto& location = aquanconKeyword.location();
                                 auto msg = fmt::format("Problem with keyword {}\n"
                                                        "In {} line {} \n"
                                                        "Connection to inactive cell ({},{},{}) is ignored", location.keyword, location.filename, location.lineno, i+1, j+1, k+1);
-                                OpmLog::warning(msg);
+                                OpmLog::warning("AQUANCON_INACTIVE_CELL", msg);
                             }
                         }
                     }
                 }
             }
+            if (num_connections_to_inactive_cells > 0) {
+                const auto& location = aquanconKeyword.location();
+                auto msg = fmt::format("Problem with keyword {}\n"
+                                       "In {} line {} \n"
+                                       "{} connections to inactive cells are ignored",
```

Comment:
Tiny nit here: If `num == 1` then this message will be
```
1 connections to inactive cells are ignored
```
which does at least look a little funny.  Can we make the message singular in that case?

Reply:
I guess I will make it so that the message only appears when there is > 1 inactive connection. Even better I think!

Reply:
> I guess I will make it so that the message only appears when there is > 1 inactive connection. Even better I think!

Agreed.  One **possible** future enhancement would be to make the message contingent on the active limit too, but `OpmLog` currently does not have a way to retrieve that value so `> 1` is a good compromise.